app/[locale]/admin/guidance/groups/[groupId]/page.tsx

```diff
@@ -1,96 +1,506 @@
+      const latestVersion = guidanceGroupData.guidanceGroup.versionedGuidanceGroup?.[0];
+      const isLatestVersionActive = latestVersion?.active ?? false;
+
+      // Publish status is determined based on isDirty and latestPublishedDate, and on whether the latest version of versionedGuidanceGroup is active
+      const currentStatus = guidanceGroupData.guidanceGroup.isDirty && guidanceGroupData.guidanceGroup.latestPublishedDate
+        ? t('status.unpublishedChanges') // Unpublished changes
+        : !guidanceGroupData.guidanceGroup.latestPublishedDate || !isLatestVersionActive
+          ? t('status.draft')
+          : t('status.published');
+
+      const transformedGuidanceGroup: GuidanceGroup = {
+        guidanceGroupId: Number(guidanceGroupData.guidanceGroup?.id),
+        name: guidanceGroupData.guidanceGroup.name || 'Untitled Guidance Group',
+        description: guidanceGroupData.guidanceGroup.description || '',
+        optionalSubset: guidanceGroupData.guidanceGroup.optionalSubset || false,
+        bestPractice: guidanceGroupData.guidanceGroup.bestPractice || false,
+        status: currentStatus,
+        lastPublishedDate: guidanceGroupData.guidanceGroup.latestPublishedDate ? formatDate(guidanceGroupData.guidanceGroup.latestPublishedDate) : '',
+      }
+
+      setGuidanceGroup(transformedGuidanceGroup);
+    };
+  }, [guidanceGroupData]);
+
+  // Intercept browser back/forward navigation to warn user of unsaved changes
+  useEffect(() => {
+    if (!hasUnsavedChanges) return;
+
+    const handlePopState = () => {
+      const confirmLeave = window.confirm(
+        Global("messaging.unsavedChangesWarning"));
+
+      if (!confirmLeave) {
+        // Push current state back to prevent navigation
+        window.history.pushState(null, '', window.location.href);
+      }
+    };
+
+    // Push a state to enable popstate detection
+    window.history.pushState(null, '', window.location.href);
+    window.addEventListener('popstate', handlePopState);
+
+    return () => {
+      window.removeEventListener('popstate', handlePopState);
+    };
+  }, [hasUnsavedChanges, Global]);
+
+  // Warn user of unsaved changes if they try to leave the page if they click on links or refresh/close the tab
   useEffect(() => {
-    // Set fake data on component mount
-    setGuidanceGroup(fakeGuidanceGroup);
-    setGuidanceTexts(fakeGuidanceTexts);
-  }, [groupId]);
+    const handleBeforeUnload = (e: BeforeUnloadEvent) => {
+      if (hasUnsavedChanges) {
+        e.preventDefault();
+        e.returnValue = ''; // Required for Chrome/Firefox to show the confirm dialog
+      }
+    };
+
+    window.addEventListener('beforeunload', handleBeforeUnload);
+    return () => {
+      window.removeEventListener('beforeunload', handleBeforeUnload);
+    };
+  }, [hasUnsavedChanges]);
+
+
+  // Determine button disabled states
+  const isUnpublishUnavailable =
+    !guidanceGroup || guidanceGroup.status === t('status.draft');
```

Comment:
is using the translated string ok here? Will it still work if the user is using `pt-BR`? There are a few other places where this is happening but I wasn't sure if it was referencing the value of the GuidanceGroup object or a page element